scripts/test.backend.sh

```diff
@@ -62,4 +62,4 @@ export POCKET_IC_MUTE_SERVER=""
 # Run tests
 
 echo "Running backend integration tests."
-cargo test -p backend
\ No newline at end of file
+cargo test -p backend "${@}"
```

Comment:
Note: `"${@}"` is an array of all arguments.  So:
* If there are no arguments, this is a noop.
* A command such as `./scripts/test.backend.sh a b c` changes the above to `cargo test -p backend a b c`
* Spaces and escapes are handled correctly:
  `./scripts/test.backend.sh --frozen "a b c"` changes the above to
  `cargo test -p backend --frozen "a b c"`